internal/view/js/page/setting.js

```diff
@@ -155,11 +178,23 @@ export default {
 						this.showErrorDialog("Password does not match");
 						return;
 					}
+                    const defaultconfig = {
```

Comment:
Is there a way we won't provide a default configuration directly from the backend? With `StructScan`, even if you have `{}` in the database, we will return the empty struct as default (everything to false.)